standards/SPIFFE_Federation.md

```diff
@@ -307,6 +307,56 @@ As described in the [Trustworthiness of the Bundle Endpoint Server](#74-trustwor
 
 Finally, it should be noted that the "links" in this chain are formed by individual HTTPS request operations (against the various bundle endpoint servers in the chain), and these operations are likely occurring at different times. SPIFFE bundle endpoint clients should be logging these HTTPS request operations, and administrators should take care to preserve these logs for future forensic analysis if necessary.
 
+### 7.7 Processing claims asserted by foreign trust domains
+
+SPIFFE Federation allows workloads in one trust domain to authenticate SVIDs, and all the claims therein, issued by other trust domains. By an "authentic" claim, we mean specifically that it can be safely assumed that the foreign trust domain authority _asserted_ the claim. This is not the same thing as saying that the claim is _true_, or that it is _correctly interpreted_.
```

Comment:
```suggestion
SPIFFE Federation allows workloads in one trust domain to authenticate SVIDs, and all the claims therein, issued by other trust domains. An "authenticated" claim means specifically that it can be safely assumed that the foreign trust domain authority _asserted_ the claim. This is not the same thing as saying that the claim was _true_ when asserted by the authority, or that it will be _correctly interpreted_.
```

Middle sentence still needs some work, I ran out of steam.

Reply:
I'm curious about authentic -> authenticated, @justinburke , can you say something about why you're proposing that change?

Reply:
In my mind, the verb authenticate means to prove something is authentic. Elsewhere in this document we use authenticated as a past-tense verb, not as an adjective.

"authenticated claim" is also not used anywhere else in the document, and the intention is to sharply define "authentic" rather than "authentic claim".